proto/stat.proto

```diff
@@ -115,34 +126,22 @@ message GetStatsResponse {
   string payload = 2;
 }
 
-// Request for GetStat API.
+// Request for GetStatSetSeries API.
 // TODO(shifucun): Deprecate GetStats() once all clients are switched to use
 // this API.
```

Comment:
TODO still relevant?

Reply:
Yep, website still uses GetStats(), need to do the migration there